api/src/test/java/org/pmiops/workbench/api/CdrVersionsControllerTest.java

```diff
@@ -1,265 +0,0 @@
-package org.pmiops.workbench.api;
```

Comment:
Renamed as CdrVersionServiceTest because the controller's functionality now lives in the service